kubernetes-maven-plugin/doc/src/main/asciidoc/inc/_access.adoc

```diff
@@ -4,13 +4,11 @@
 
 == Docker Access
 
-WARNING: This section is work-in-progress and not yet finished
-
-For {cluster} builds the {plugin} uses the Docker remote API so the URL of your Docker Daemon must be specified. The URL can be specified by the dockerHost or machine configuration, or by the `DOCKER_HOST` environment variable. If not given
+For {cluster} builds the {plugin} uses the Docker remote API so the URL of your Docker Daemon must be specified. The URL can be specified by the dockerHost or by the `DOCKER_HOST` environment variable.
 
 The Docker remote API supports communication via SSL and
 authentication with certificates.  The path to the certificates can
-be specified by the certPath or machine configuration, or by the
+be specified by the certPath or by the
```

Comment:
Don't we no longer support Docker-Machine in Windows?

Reply:
I checked on the internet about this Docker Machine and it looks like an alternative to Kubernetes. In my opinion, it didn't make sense to keep this parameter

Reply:
:laughing: You've lived a great life without Windows then ;)

Docker machine is a virtual machine that is launched in Windows systems (Home) (I think this is the same for macOS). It's an actual virtual virtual machine running Linux where Docker is hosted. Then the Docker daemon host is shared with windows. (Maybe check this [(translated) article](https://translate.google.com/translate?sl=auto&tl=en&u=https%3A%2F%2Fblog.marcnuri.com%2Fdocker-instalando-docker-en-windows%2F) I wrote way back in 2015 so you understand what this is)

You are probably referring to Docker Swarm when talking about some sort of replacement for Kubernetes, right?

Reply:
I got this from this article: https://stackshare.io/stackups/docker-machine-vs-kubernetes

Reply:
ohk, let me add it back.

Reply:
I can't see any article, it's just a comparison tool. I added Google Drive there too for completeness ;P

https://stackshare.io/stackups/docker-machine-vs-kubernetes-vs-google-drive

Reply:
oh, looks like I shouldn't trust random stuff on internet

Reply:
oh, looks like I shouldn't trust ~random~ stuff on internet